src/Spd.Manager.Licence/BizProfileValidation.cs

```diff
@@ -1,4 +1,5 @@
 using FluentValidation;
+using Microsoft.IdentityModel.Tokens;
```

Comment:
why we need this?

Reply:
it is used by the method `IsNullOrEmpty()`